File: storage/storage-mysql/src/main/java/io/say/better/storage/mysql/domain/constant/ProgressState.java
package io.say.better.storage.mysql.domain.constant;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum ProgressState {
	READY("수업 가능"),
	PROCESSING("수업 진행"),
	HAVE_TO_REVIEW("수업 리뷰"),
	TERMINATED("수업 종료");

	private final String description;
}

File: storage/storage-mysql-kt/src/main/kotlin/io/say/better/storage/mysql/domain/constant/ProgressState.kt
package io.say.better.storage.mysql.domain.constant

enum class ProgressState(description: String) {
    READY("수업 가능"),
    PROCESSING("수업 진행"),
    HAVE_TO_REVIEW("수업 리뷰"),
    TERMINATED("수업 종료");
}
